launcher/image/preload.sh

```diff
@@ -2,11 +2,19 @@
 
 readonly OEM_PATH='/usr/share/oem'
 readonly CS_PATH="${OEM_PATH}/confidential_space"
+readonly EXPERIMENTS_SCRIPT="experiments_sync"
+readonly GCS_BUCKET="gs://confidential-space-images_third-party_test"
 
 copy_launcher() {
   cp launcher "${CS_PATH}/cs_container_launcher"
 }
 
+copy_exeriment_client() {
+  gcloud storage cp ${GCS_BUCKET} "${CS_PATH}/${EXPERIMENTS_SCRIPT}"
+  chmod +x "${CS_PATH}/${EXPERIMENTS_SCRIPT}"
```

Comment:
I think this may be why the script isn't running: it should go after the cp.

Reply:
Fixed this previously, thanks for catching this.